src/hooks.server.ts

```
@@ -84,6 +84,11 @@ function isRateLimited(
 	const key = ip;
 	const entry = rateLimitStore.get(key);
 
+	//Skip rate limiting for health checks
+	if (path === '/api/health') {
```

ESLint

[error] 'path' is not defined.

(no-undef)